src/journey-maps/angular/util/input-validation.ts

```diff
@@ -3,7 +3,9 @@ import { SbbJourneyMapsRoutingOptions } from '../journey-maps.interfaces';
 export const getInvalidRoutingOptionCombination = (
   routingOptions: SbbJourneyMapsRoutingOptions
 ): string[] => {
-  const nonEmptyOptions = Object.entries(routingOptions).filter(([_, value]) => value);
+  const nonEmptyOptions = Object.entries(routingOptions).filter(
+    ([_, value]) => value && _ !== 'railNetworkOptions'
```

Comment:
Hmmm... Hacky... Diese Methode wird immer schlimmer.

Reply:
Ja, das stimmt. Könnte wir mal anders bauen. Hast du eine Idee wie?

Reply:
Nein... Es ist für mich auch kein Blocker für's Mergen.

Ich habe mir höchstens überlegt, ob wir `railNetworkOptions` in `SbbStyleOptions` verschieben wollen.

Aber wenn wir sagen, dass die Farbe vom Streckennetz mit einer Route zusammenhängen soll, dann passt das auch nicht wirklich. (Oder wir trennen es explizit und der User muss das selber steuern)